### ISSUE TITLE
[Accessibility] Missing <html> lang attribute on landing page

### ISSUE THREAD

**Issue by bbambozzi** [quarkusio/quarkusio.github.io#2022]
**Describe the bug**
The landing page's <html\> tag is missing its [lang attribute.](https://dequeuniversity.com/rules/axe/4.9/html-has-lang)
We should add one to make browsing easier for folks using screen readers.

**Comment by bbambozzi** [comment]
Fixed by #2023
